### DISCUSSION TITLE
Visibility of project budget for User Role

### DISCUSSION THREAD

**Discussion by stevenrenaud** [kimai/kimai#3609] [Questions & Answers]
Hello,

Kimai is awesome.

I have a very little concern, in the report  "Monthly Report", the "Revenue" of the project is visible for the role user.
Is there any way to hide this data for the role "user"  in another way than disallow all the report section ?

![kimai_monthly_report_with_revenue](https://user-images.githubusercontent.com/50968243/199003513-da14b4f0-1e99-4de2-8d37-d389753d2e05.png)

In advance thank you.

**Comment by kevinpapst** [comment]
Next time, please create a bug report if something obvious is wrong!
Here it is: https://github.com/kevinpapst/kimai2/issues/3626

Will be fixed with the next version.

**Reply by kevinpapst** [reply]
Oh and **Thank you** for reporting 👍
